### ISSUE TITLE
WEBP Support

### ISSUE THREAD

**Issue by IZeeLikeEyeZee** [bimbo-garden/greenhouse#7]
Support for WEBP files for emotes. Would be good to have an option to toggle to gif instead when viewing emotes on the website tho bc makes it easier for people to download emotes since discord and stuff like that doesn't support WEBP (yet at least). 7tv uses AVIF so we could try that too maybe but ngl I dont see the benefit since it doesn't save all that much space and is barely supported by anything atm from what I could tell... I'll leave that one up to y'all since it doesn't have any benefit for making emotes, but just has to do with technical shit.

**Comment by FFY00** [comment]
Yeah, WEBP should be the main format we work with. We can make a GIF version of emotes available to download on the emote page.

**Comment by IZeeLikeEyeZee** [comment]
That's perfect, also PNGs for static emotes ofc

**Comment by IZeeLikeEyeZee** [comment]
adding avif format here because I just looked into it some more and while it has less support than webp/gif, the file sizes are insanely lower and quality is better
